messages/md.retrieve.json

```diff
@@ -0,0 +1,48 @@
+    "examples": [
+      "$ sfdx force:mdapi:beta:retrieve:report",
+      "$ sfdx force:mdapi:beta:retrieve:report -r path/to/retrieve/dir -w 15",
+      "$ sfdx force:mdapi:beta:retrieve:report -r path/to/retrieve/dir --unzip --zipfilename apexClasses.zip"
+    ]
+  },
+  "flags": {
+    "retrievetargetdir": "directory root for the retrieved files",
+    "unpackaged": "file path of manifest of components to retrieve",
+    "sourcedir": "source dir to use instead of the default package dir in sfdx-project.json",
+    "packagenames": "a comma-separated list of packages to retrieve",
+    "singlepackage": "indicates that the zip file points to a directory structure for a single package",
+    "zipfilename": "file name to use for the retrieved zip file",
+    "unzip": "extract all files from the retrieved zip file",
+    "wait": "wait time for command to finish in minutes (default: -1 (no limit))",
+    "apiversion": "target API version for the retrieve",
+    "verbose": "verbose output of retrieve result",
+    "jobid": "job ID of the retrieve you want to check; defaults to your most recent CLI retrieval if not specified"
+  },
+  "flagsLong": {
+    "retrievetargetdir": "The root of the directory structure where the retrieved .zip or metadata files are put.",
+    "unpackaged": "The complete path for the manifest file that specifies the components to retrieve.",
+    "sourcedir": "The source directory to use instead of the default package directory specified in sfdx-project.json",
+    "packagenames": "A comma-separated list of package names to retrieve.",
+    "singlepackage": "Indicates that the specified .zip file points to a directory structure for a single package. By default, the CLI assumes the directory is structured for a set of packages.",
+    "zipfilename": "The file name to use for the retrieved zip file.",
+    "unzip": "Extract all files from the retrieved zip file.",
+    "wait": "The number of minutes to wait for the command to complete.",
+    "apiversion": "Use to override the default, which is the latest version supported by your CLI plug-in, with the version in your package.xml file.",
+    "verbose": "Indicates that you want verbose output from the retrieve operation.",
+    "jobid": "The job ID (asyncId) of the retrieve you want to check. If not specified, the default value is the ID of the most recent metadata retrieval you ran using Salesforce CLI. You must specify a --retrievetargetdir. Use with --wait to resume waiting."
+  },
+  "InvalidFlagPath": "The %s command parameter specifies an invalid path: %s\n%s",
+  "InvalidPackageNames": "You specified [%s]. Try again and specify only one package when using --singlepackage.",
+  "MissingRetrieveId": "The jobid command parameter was not provided, neither directly nor from a previous retrieval.",
+  "checkStatus": "To check the status of this retrieve, run \"sfdx force:mdapi:beta:retrieve:report %s\""
```

Comment:
I think it's clearer to indicate that running the report command will
1. check the status
2. but also, if the retrieve finished, store the output in your target dir, maybe unzipped if you said to

Reply:
I'll update, thanks.